pvoutput/utils.py

```diff
@@ -1,5 +1,5 @@
 """ Utilities """
-
+import re
```

Comment:
This is not being used

Reply:
correct, removed, outdated in the mean time